### ISSUE TITLE
Example workflow - why use niche custom nodes in example workflows?

### ISSUE THREAD

**Issue by quasiblob** [Lightricks/ComfyUI-LTXVideo#83]
Hi.

Could you please change the default Text-to-video example?

I don't like to usually complain about free things, but the "Flux Prompt Enhance" node isn't very popular. It only has 62 stars:

![image](https://github.com/user-attachments/assets/04a9878e-2be8-45f5-8326-b4748eed1497)

Now, low star count does NOT mean this extension could somehow be malicious (because it is not very popular), but the repo of this custom node also says this:

![image](https://github.com/user-attachments/assets/c234336f-f0aa-4fe8-becc-98d9655b6aef)

So basically seems like the author has stopped also developing it.

**Comment by Packsod** [comment]
Rather than complaining, I can give some suggestions. The sample workflow already uses String Function 🐍, which supports regular expressions, so no additional nodes are needed for replacing specific words .
![image](https://github.com/user-attachments/assets/5bd6c023-5946-4ea4-8fee-e11c38c9c569)

**Comment by quasiblob** [comment]
"I can give some suggestions." - thanks but it is not like I can't replace it  (although in this case wouldn't bother to start fiddling with regex myself, as I barely remember anything about regex lol) I've installed too many custom nodes already. This seems to be a common trend with shared workflows on several sites. If it would be possible to avoid using some less popular nodes, then why not use the popular one instead? Even better, don't use any unnecessary custom nodes if possible, when sharing example workflows.

**Comment by Packsod** [comment]
This advice is for the repo developers, they definitely have a reason to use these nodes, the pysssss nodes are almost a must. I'm just saying the examples could be improved a bit.
